src/config.c

```diff
@@ -564,15 +564,15 @@ void capture_comment_str(const char *com, int lineno) {
 
 // Capture assignments (keywords left of =) and associate comments to them
 void capture_lvalue_kw(const char *kw, int lineno) {
-  if(!strcmp(kw, "memory")) {   // Push part comments and start memory comments
+  if(str_eq(kw, "memory")) {   // Push part comments and start memory comments
```

Comment:
This section lines up post-command comments at column 32 (counting from 0). Please move comment one to right.

Reply:
Done, thanks!